src/Installers/Windows/Common/dotnethome_x64.wxs

```diff
@@ -5,23 +5,28 @@
     <?define Platform = "$(sys.BUILDARCH)"?>
   <?endif?>
 
-  <!-- InstallerArchitecture matches the expected values for PROCESSOR_ARCHITECTURE
-       https://docs.microsoft.com/en-us/windows/win32/winprog64/wow64-implementation-details  -->
+  <!-- InstallerNativeMachine matches the expected values for image file machine constants
+       https://docs.microsoft.com/en-us/windows/win32/sysinfo/image-file-machine-constants -->
   <?if $(var.Platform)~=x86?>
-    <?define InstallerArchitecture="X86"?>
+    <?define InstallerNativeMachine=332?>
```

Comment:
332 = 0x14c = IMAGE_FILE_MACHINE_I386

@dougbu this file is a copy-paste from arcade, at the moment I'd prefer to keep it identical rather than change these.  The hyperlink in above gives the link that helps folks decode these, I don't think it's any more readable to duplicate parts of that document.

Reply:
FWIW we had a similar discussion here https://github.com/wixtoolset/wix3/pull/538#discussion_r701179652

Reply:
I guess&hellip; My thought was the decimal numbers here and the hex numbers in the document made the combination hard to use. For example, I copied the wrong entry 😄

In any case, let's get this in and postpone adding comments like `332 = 0x14c = IMAGE_FILE_MACHINE_I386`.

Reply:
I will apply this feedback into arcade/7.0 and see about getting rid of the duplication moving forward when I'm not on a time crunch for 6.0 / servicing.